src/command.rs

```diff
@@ -43,6 +47,85 @@ pub enum Command<'l> {
     SendRemaining,
     /// Get Credential data
     GetCredential(GetCredential<'l>),
+
+    YKSerial,
+    YKGetStatus,
+    YKGetHMAC(YKGetHMAC<'l>),
```

Comment:
Standard rust naming scheme would be  `YkSerial`, YkGetStatus` and `YkGetHmac`:

https://rust-lang.github.io/api-guidelines/naming.html

> In `UpperCamelCase`, acronyms and contractions of compound words count as one word: use `Uuid` rather than `UUID`, `Usize` rather than `USize` or `Stdin` rather than `StdIn`.

Reply:
I've made an exception here, but I presume this will show up in the linting tools forever. 
To be corrected to `YkGetHmac`.